components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -110,12 +100,22 @@ export default function OrganizationSelector() {
             // collaborator can read org setting via API so that other feature like restrict org workspace classes could work
             // we only hide the menu from dashboard
             linkEntries.push({
-                title: "Settings",
-                customContent: <LinkEntry>Settings</LinkEntry>,
+                title: "Organization Settings",
+                customContent: <LinkEntry>Organization Settings</LinkEntry>,
                 active: false,
                 separator: false,
                 link: "/settings",
             });
+
+            if (configurationsAndPrebuilds) {
+                linkEntries.push({
+                    title: "Repository settings",
+                    customContent: <LinkEntry>Repository settings</LinkEntry>,
+                    active: false,
+                    separator: false,
+                    link: "/repositories",
+                });
+            }
```

Comment:
**suggestion**: I'd always place the settings at the bottom here, what made us move this from the top?

Reply:
this is partly a mistake, the menu item was supposed to go above the org settings. i replied about the general move on Linear [[1](https://linear.app/gitpod/issue/EXP-1357/call-it-repositories#comment-79cfd6e7)]

Reply:
Ah, ok as long as the settings link is that the bottom, the rest can be ok. Sorry for the interruption! ⚡